### ISSUE TITLE
Plugin installation not working

### ISSUE THREAD

**Issue by ghost** [SpaceCP/Bugs#216]
So, I`m trying to add the 'Towny' plugin to the server. But when I press the add button it just give me de notification at the bottom and nothing happens. 

http://palmergames.com/file-repo/Towny%20Advanced/Development/0.91.0.2/Towny.jar <-- Plugin file
http://prntscr.com/b4umlr <-- Result

**Comment by Willsr71** [comment]
More info:
![image](https://i.gyazo.com/1c17f8f70c5cde484565b6efb34307e5.png)

**Comment by tlaundal** [comment]
Yeah, the empty notification is something we know about. It's fixed in a
branch we're developing on, should be pushed sometime.

Your server console should show an error, though. Have a look there, for
example by clicking the console icon in the top right corner of SpaceCP
when viewing your server.

**Comment by ghost** [comment]
I've looked at my console and it doesn't show anything except the usual minecraft logs. But the modal is not dissapearing either I think it's just not doing anything back-end wise.

**Comment by ghost** [comment]
After some more trail and error I have found out I can't install any plugin.
